app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt

```diff
@@ -186,26 +185,21 @@ class AddProfileActivityTest {
   }
 
   @Test
-  fun testAddProfileActivity_inputName_hasPrefixTextRequired_IsDisplayedOnFocus() {
+  fun testAddProfileActivity_inputName_hasHelperTextRequired_IsDisplayed() {
```

Comment:
```suggestion
  fun testAddProfileActivity_inputName_hasRequiredHelperTextDisplayed() {
```

Ditto for the other test names.

Reply:
Done. The test cases are renamed with the latest push. PTAL